internal/nodes/dequeue_ingestion.go

```diff
@@ -120,7 +120,7 @@ func DequeueIngestion(key string, fullsync bool) {
 	}
 
 	// handle the services APIs
-	if lib.GetAdvancedL4() {
+	if lib.GetAdvancedL4() || lib.UseServicesAPI() && (objType == utils.L4LBService || objType == lib.Gateway || objType == lib.GatewayClass) {
```

Comment:
It's how the project has evolved over time. utils used to be in container-lib which was a different project on GitHub but it has been since merged into AKO.